octopoes/tools/rename-origin-method.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python
+
+import json
+import logging
+from typing import Any
+
+import click
+from xtdb_client import XTDBClient
+
+logger = logging.getLogger(__name__)
+
+
+@click.group(
+    context_settings={
+        "help_option_names": ["-h", "--help"],
+        "max_content_width": 120,
+        "show_default": True,
+    }
+)
+@click.option("-n", "--node", default="0", help="XTDB node")
+@click.option(
+    "-u",
+    "--url",
+    default="http://localhost:3000",
+    help="XTDB server base url",
+)
```

Comment:
Wouldn't defaults based on the settings (i.e. the env) make more sense so this works out of the box for most non-dev installations?

Reply:
We could use the env file to find the XTDB instance

Reply:
We can just tell click to use the use the `XTDB_URI` env var.

```suggestion
@click.option("-n", "--node", default="0", help="XTDB node")
@click.option(
    "-u",
    "--url",
    default="http://localhost:3000",
    envvar="XTDB_URI",
    help="XTDB server base url",
)
```